src/diffusers/configuration_utils.py

```diff
@@ -377,8 +474,33 @@ def __repr__(self):
 
     @property
     def config(self) -> Dict[str, Any]:
+        """
+        Returns the config of the class as a frozen dictionary
+
+        Returns:
+            `Dict[str, Any]`: Config of the class.
+        """
         return self._internal_dict
 
+    @property
+    def compatibles(self):
+        """
+        Returns all classes that are compatible with this configuration
+
+        Returns:
+            `List[ConfigMixin]`: List of compatible classes
+        """
+        return self._get_compatibles()
+
+    @classmethod
+    def _get_compatibles(cls):
+        compatible_classes_str = list(set([cls.__name__] + cls._compatible_classes))
```

Comment:
I'm a bit lost here, is `_compatible_classes` defined for classes that are not schedulers?

Reply:
Fair point, will move to `SchedulerMixin` - guess it makes less sense for models